src/components/query-viewer-chart/controller/highcharts-options.ts

```diff
@@ -1524,7 +1524,7 @@ function groupPoints(
     dateStr: null,
     name: null
   };
-  let pointAdd: { x: string; y: number; name: string };
+  let pointAdd: { x: string; y: GxBigNumber; name: string };
```

Comment:
`[groupPoints]` It would be nice to add some tests using this utility with different values to validates the new behavior.